### PR TITLE
Update to RSpec 3.2

### PR DESCRIPTION
**Pull request by AgentAntelope**
Thanks to @bobknoths for prompting this :)

### CODE REVIEW

File: .rspec [hunk 1]
```diff
@@ -1,3 +1,3 @@
 --color
 --format progress
---default_path ./
+--default-path ./
```
Comment (malpinder) [inline .rspec]: What's actually changed on this line?

File: spool_2/regex_spec.rb [hunk 2]
```diff
@@ -4,73 +4,74 @@
 
   let(:phrase) { "It was a bright cold day in April, and the clocks were striking thirteen." }
 
-  it "matches using =~" do
+  it "matches using match" do
```
Comment (malpinder) [inline spool_2/regex_spec.rb]: What's the reasoning for changing `=~` to `match`? Is the former deprecated in this Rspec version?

File: spool_2/regex_spec.rb [hunk 3]
```diff
@@ -4,73 +4,74 @@
...
     expect(phrase).to __ /bright/
   end
+
   it "matches using .match" do
-    expect(/April/.__(phrase)).to be_true
+    expect(/April/.__(phrase)).to be_truthy
   end
 
-  # What's the difference between using =~ and .match?
+  # What's the difference between using match and .match?
```
Comment (malpinder) [inline spool_2/regex_spec.rb]: What is the difference now? `=~` doesn't work the same as `.match`, but IIRC `.match` does work the same as the `match` matcher.

File: spool_1/boolean_spec.rb [hunk 4]
```diff
@@ -18,34 +18,34 @@
 
   describe "falsiness" do
     it "is false for false" do
-      expect(__).to be_false
+      expect(__).to be_falsey
```
Comment (malpinder) [inline spool_1/boolean_spec.rb]: :+1:

File: .rspec (hunk 1 above)
Comment (AgentAntelope) [inline .rspec]: s/_/- :)

File: spool_2/regex_spec.rb (hunk 2 above)
Comment (AgentAntelope) [inline spool_2/regex_spec.rb]: Not deprecated, but it doesn't work with `#to`

File: spool_1/boolean_spec.rb (hunk 4 above)
Comment (AgentAntelope) [inline spool_1/boolean_spec.rb]: I think this is equally as clear as it used to be? Not sure what the comment would need to be like.

File: spool_2/regex_spec.rb (hunk 3 above)
Comment (AgentAntelope) [inline spool_2/regex_spec.rb]: The problem with using find and replace I guess. I think this comment isn't relevant now.

File: spool_1/boolean_spec.rb (hunk 4 above)
Comment (malpinder) [inline spool_1/boolean_spec.rb]: What I mean is, the test description says `false for false` when it is actually `falsey for false`.

File: spool_1/numbers_spec.rb [hunk 5]
```diff
@@ -30,7 +30,7 @@
     end
     it "can be smaller than 1" do
       float = __
-      expect(float).to < 1
+      expect(float < 1).to eq true
```
Comment (malpinder) [inline spool_1/numbers_spec.rb]: I'd prefer to use `be` in this case: `expect(float).to be < 1`

File: spool_2/regex_spec.rb (hunk 2 above)
Comment (malpinder) [inline spool_2/regex_spec.rb]: Okay. I don't think the differences between `=~` and `match` affect any of these specs so it should be fine.

File: spool_3/kitten_spec.rb [hunk 6]
```diff
@@ -18,31 +18,31 @@ class Kitten
 
   describe "feed" do
     it "returns true when fed ketchup on fish" do
-      expect(kitten.feed('ketchup on fish')).to be_true
+      expect(kitten.feed('ketchup on fish')).to be_truthy
```
Comment (malpinder) [inline spool_3/kitten_spec.rb]: These specs all still want to `be_true` and `be_false` - I'm explicitly wanting the implementation of `feed` to return a boolean.

**Pull request merged**